mne/viz/topo.py

```diff
@@ -582,13 +584,13 @@ def set_ylim(x):
         else:
             ylim_ = zip(*[np.array(yl) for yl in ylim_])
     else:
-        raise ValueError('ylim must be None ore a dict')
+        raise ValueError('ylim must be None or a dict')
```

Comment:
This should be `TypeError` and we try to do ``got %s' % type(ylim)` for these. (I know it's not related to the PR, but might as well sneak it in since you're in there.)